pkg/drivers/kic/kic.go

```diff
@@ -292,40 +298,43 @@ func (d *Driver) Remove() error {
 func (d *Driver) Restart() error {
 	s, err := d.GetState()
 	if err != nil {
-		return errors.Wrap(err, "get kic state")
+		glog.Warningf("get state during restart: %v", err)
```

Comment:
I'd make it clear this is an error, "error getting state during restart: %v"

Reply:
will do